src/components/LoginRegisterPanel/LoginRegisterPanel.tsx

```diff
@@ -1,3 +1,4 @@
+
 import { useState } from 'react';
 import { IoCloseCircleOutline } from 'react-icons/io5';
 import { FaRegEye } from 'react-icons/fa';
```

Comment:
states from line 11-13 it's great option to learn how rexus works, and combine it together